core/src/main/java/fr/sncf/osrd/infra/api/tracks/undirected/TrackEdge.java

```diff
@@ -42,5 +42,5 @@ public sealed interface TrackEdge permits SwitchBranch, TrackSection {
     ImmutableRangeMap<Double, LoadingGaugeConstraint> getLoadingGaugeConstraints();
 
     /** Returns a set of voltages usable at any position */
-    RangeMap<Double, Set<Integer>> getVoltages();
+    RangeMap<Double, String> getVoltages();
```

Comment:
How does this work? Don't we need a set of values? If we don't, the comment needs to be updated

Reply:
We don't need a set since you cannot have multiple voltages at the same infra location. This would require having several physical catenaries at the same location.